dev-python/pyside-tools/pyside-tools-9999.ebuild

```diff
@@ -0,0 +1,118 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+#FIXME: Restore Python 2.7 support after this upstream issue is resolved:
+#    https://bugreports.qt.io/browse/PYSIDE-508
+#Even after this issue is resolved, further ebuild changes will probably be
+#needed to fully support Python 2.7. Unlike PySide2 and Shiboken2, the root
+#"CMakeLists.txt" file in pyside2-tools provides no ${PYTHON_EXECUTABLE} option.
+#Instead, the ${PYTHON_EXTENSION_SUFFIX} and ${PYTHON_BASENAME} options must be
+#passed with values specific to Python 2.7. Additionally, note that the
+#"python2.7-config" command provides no "--extension-suffix" option.
+PYTHON_COMPAT=( python3_{4,5,6} )
+# PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+
+inherit cmake-utils python-r1 virtualx git-r3
+
+DESCRIPTION="PySide development tools (lupdate, rcc, uic)"
+HOMEPAGE="https://wiki.qt.io/PySide2"
+EGIT_REPO_URI=(
+	"git://code.qt.io/pyside/${PN}.git"
+	"https://code.qt.io/git/pyside/${PN}.git"
+)
+#FIXME: Switch to the clang-enabled "dev" branch once stable.
+EGIT_BRANCH="5.6"
+
+# See the top-level "LICENSE-rcc" and "LICENSE-uic" files for licensing.
+LICENSE="BSD GPL-2"
+SLOT="2"
+KEYWORDS=""
+IUSE="test"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+# Minimum version of Qt required.
+QT_PV="5.6*:5"
+
+RDEPEND="
+	${PYTHON_DEPS}
+	dev-python/pyside:2[gui,${PYTHON_USEDEP}]
+	=dev-qt/qtcore-${QT_PV}
+	=dev-qt/qtgui-${QT_PV}
+"
+DEPEND="${RDEPEND}
+	test? ( virtual/pkgconfig )
+"
+
+src_prepare() {
+	cmake-utils_src_prepare
+
+	preparation() {
+		if python_is_python3; then
```

Comment:
This whole conditional works in `pyside-tools-0.2.15-r1` because we duplicate the sources for each python impl (`python_copy_sources`) and we do an in-source build (`CMAKE_IN_SOURCE_BUILD="1"`). Neither condition is true in this ebuild, so I doubt it'll work. You might not have noticed the problem because you only ever built it for one python3 impl.